envision/server.py

```diff
@@ -300,7 +306,16 @@ def on_close(self):
 
     async def on_message(self, message):
         """Asynchronously receive messages from the Envision client."""
-        frame_time = next(ijson.items(message, "frame_time", use_float=True))
+        it = ijson.parse(message)
+        frame_time = None
+        # find the first number value. It is the frame time.
```

Comment:
This message is a bit misleading since you set the last value in `it` array which is a number in the to be `frame_time`.

Reply:
Yes, although, `it` and `It` are obviously different. 🤣